rmw_implementation/CMakeLists.txt

```diff
@@ -91,8 +78,6 @@ register_rmw_implementation(
   "cpp:rosidl_typesupport_cpp"
 )
 
-# necessary since get_available_rmw_implementations excludes rmw_implementation
-list(APPEND RMW_IMPLEMENTATIONS "rmw_implementation")
```

Comment:
Can you describe why this was removed?

Reply:
It's not needed since `rmw-implementation-extras.cmake.in` no longer references `@RMW_IMPLEMENTATIONS@`